test/e2e/framework/pod/wait.go

```diff
@@ -53,14 +53,10 @@ const (
 	podScheduledBeforeTimeout = podListTimeout + (20 * time.Second)
 
 	// podStartTimeout is how long to wait for the pod to be started.
-	// Initial pod start can be delayed O(minutes) by slow docker pulls.
-	// TODO: Make this 30 seconds once #4566 is resolved.
-	podStartTimeout = 5 * time.Minute
+	podStartTimeout = 30 * time.Second
```

Comment:
(30 * time.Second) this seems will result in timeout now

Reply:
my suggestion will be to remove the TODO
```
// Initial pod start can be delayed O(minutes) by slow docker pulls.
// TODO: Make this 30 seconds once #4566 is resolved.
```

and keep the podStartTimeout to 5 minutes.
this constant seems to be used in a lot of places, so let's not touch it for now.
[1]

Reply:
get it . thanks